graylog2-web-interface/src/components/widgets/Widget.jsx

```diff
@@ -1,21 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import ReactDOM from 'react-dom';
 import $ from 'jquery';
 import { PluginStore } from 'graylog-web-plugin/plugin';
-
```

Comment:
We tend to separate imports of modules and imports of components in the same code base by an empty line.